crates/sui-framework/sources/object.move

```diff
@@ -39,6 +39,11 @@ module sui::object {
         id: ID,
     }
 
+    // === address ===
+
+    /// Convert raw bytes into an address
+    public native fun address_from_bytes(bytes: vector<u8>): address;
```

Comment:
I'm confused... I don't see this used anywhere? Didn't realize that was the case lol

Reply:
Yep, it's only used in the calibration tests!